### PR TITLE
Register aryanpatel.is-a.dev

### PR DESCRIPTION
**Pull request by aryanpatel142006**
Register aryanpatel.is-a.dev with CNAME record pointing to cname.vercel-dns.com..

### CODE REVIEW

File: domains/aryanpatel.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "aryanpatel142006",
+           "email": "",
+           "discord": "906554993199415366",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.fa_H8QpLZ_OupaUgljIiW3sj6TLUm0JgsLuRUZSKsk2f7FVzSFrUFZ5LRG3DQXkzPYGuN3PqyZcjtAimqJyUz-p8Ab3lgYkXRigP0v2rKBx_ODE_0BceWAU911DDIjpKOY5gKzxeh-RPnamR55senjLrP1Vb0WsT6lDPZIaA1Z5_m-Go7bHvjdjLJl4i0ttKvFgt414ELXRrySrRLq417kLtXebXwKl-jJhUr5j-0hgCS7_klh7vt65KhuWA4XLAAVdwune4G88RLHcOA3_qEwc039FxsJxjUqfHTcPGQZv_dNSckiQovZPmH2hKaJ-NhF1OisYdhtTwWhRS5yzUFA.Fee5fCqO54I7O42Nbs17Vw.zyIGV1sn88_wI6PoC-GgxeuK8Mubdhtb3mRdXU8ck9RImf8f2tWdqisuK7W-DTdE69CPfQ3vZhdKHT8zni_ksuSuiIffnFL9i2sptxQ7ZZGker1B6sRWLekL1vWDlEU-.eHRvZyJxTHtKsWk_k-XcvA"
+        },
+    
+        "record": {
+            "CNAME": "cname.vercel-dns.com."
```
Comment (Stef-00012) [inline domains/aryanpatel.json:10]: CNAME can not end with `.`
```suggestion
            "CNAME": "cname.vercel-dns.com"
```

**Review by Stef-DP — CHANGES_REQUESTED** [review]

**Comment by Stef-00012** [comment]
> Hey, so I read some of the comments in the previous PRs and turns out I'll have to finish the website, before getting the domain, and I'm a beginner I just learnt hosting a website, so I'm creating my portfolio rn, so it's ok if you just close the PR, I really like it cause I'm getting my full name in the subdomain, which I never get anywhere,but I'm afraid someone might take the subdomain, so it'd be great if you could merge the PR and I'll update the website ASAP, but I totally understand if you cant't :)

I don't think ur name is that common that might be taken by anyone else

**Review by iostpa — CHANGES_REQUESTED** [review]
Can you provide a preview of your website?

**Comment by CuteDog5695** [comment]
I'll leave this PR up until Monday, then I'll close it

**Pull request closed without merging**